giza/giza/content/options/tasks.py

```diff
@@ -44,8 +44,16 @@ def option_tasks(conf):
 
     tasks = []
     for dep_fn, option in o.content_iter():
+        if option.has_field('command'):
+            t = option.program.split(" ")
```

Comment:
You don't need to give split() an argument here, I don't think. By default, it will split on any whitespace.